proto/v2/v2.proto

```diff
@@ -19,11 +19,22 @@ package datacommons.v2;
 
 import "entity.proto";
 
-message PropertyValuesV2Request {
+message Arc {
+  string property = 1;
+  repeated datacommons.EntityInfo nodes = 2;
+}
+
+message QueryV2Request {
+  // Nodes to query for.
   repeated string nodes = 1;
+  // The expression for the property or the chained property.
   string property = 2;
+  // Max number of entries to be returned.
```

Comment:
What is an "entry"?  Is it nodes or triples?

Reply:
It's nodes.